scripts/make_bulk_data.sh

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
+set -e
+
+echo "Installing utilities"
+
+# Set up Sentry
+curl -sL https://sentry.io/get-cli/ | bash
+eval "$(sentry-cli bash-hook)"
+
+# Set up AWS tools and gnupg (needed for apt-key add, below)
+apt install -y awscli gnupg
+
+# Install latest version of pg_dump (else we get an error about version mismatch
+echo "deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+curl --silent 'https://www.postgresql.org/media/keys/ACCC4CF8.asc' |  apt-key add -
+apt-get update
+apt-get install postgresql-client-14
+
+# Stream to S3
+
+echo "Streaming search_court to S3"
+PGPASSWORD=$DB_PASSWORD psql \
+	--command \
+	  'set statement_timeout to 0;
+	   COPY search_court (
+	       id, pacer_court_id, pacer_has_rss_feed, fjc_court_id, date_modified,
+	       in_use, has_opinion_scraper, has_oral_argument_scraper, position,
+	       citation_string, short_name, full_name, url, start_date, end_date,
+	       jurisdiction
+	   ) TO STDOUT WITH (FORMAT csv, ENCODING utf8, HEADER)' \
+	--host $DB_HOST \
+	--username $DB_USER \
+	--dbname courtlistener | \
+	bzip2 | \
+	aws s3 cp - s3://com-courtlistener-storage/bulk-data/courts-`date -I`.csv.bz2 --acl public-read
+
+echo "Streaming search_docket to S3"
+PGPASSWORD=$DB_PASSWORD psql \
+	--command \
+	  'set statement_timeout to 0;
+	   COPY search_docket (
+	       id, date_created, date_modified, source, appeal_from_str,
+	       assigned_to_str, referred_to_str, panel_str, date_cert_granted,
+	       date_cert_denied, date_argued, date_reargued, date_reargument_denied,
+	       date_filed, date_terminated, date_last_filing, case_name_short,
+	       case_name, case_name_full, slug, docket_number, docket_number_core,
+	       pacer_case_id, cause, nature_of_suit, jury_demand, jurisdiction_type,
+	       appellate_fee_status, appellate_case_type_information, mdl_status,
+	       filepath_local, filepath_ia, filepath_ia_json, date_blocked, blocked,
+	       appeal_from_id, assigned_to_id, court_id, idb_data_id,
+	       originating_court_information_id, referred_to_id
+	   ) TO STDOUT WITH (FORMAT csv, ENCODING utf8, HEADER)' \
+	--host $DB_HOST \
+	--username $DB_USER \
+	--dbname courtlistener | \
+	bzip2 | \
+	aws s3 cp - s3://com-courtlistener-storage/bulk-data/dockets-`date -I`.csv.bz2 --acl public-read
+
+echo "Streaming search_opinioncluster to S3"
+PGPASSWORD=$DB_PASSWORD psql \
+	--command \
+	  'set statement_timeout to 0;
+	   COPY search_opinioncluster (
+	       id, date_created, date_modified, judges, date_filed,
+	       date_filed_is_approximate, slug, case_name_short, case_name,
+	       case_name_full, scdb_id, scdb_decision_direction, scdb_votes_majority,
+	       scdb_votes_minority, source, procedural_history, attorneys,
+	       nature_of_suit, posture, syllabus, headnotes, summary, disposition,
+	       history, other_dates, cross_reference, correction, citation_count,
+	       precedential_status, date_blocked, blcoked, docket_id
```

Comment:
Typo in `blocked` 

Should we include `date_blocked`? and blocked.

Reply:
Including date_blocked is fine. Nothing too interesting about it, really.